doc/jcli/transaction.md

```diff
@@ -26,9 +26,12 @@ There are also functions to help decode and display the
 content information of a transaction:
 
 * `info`
-* `id` to get the **Transaction ID** of the transaction
+* `data-for-witness` get the data to sign from a given transaction
+* `fragment-id` get the **Fragment ID** from a transaction in *sealed* state
 * `to-message` to get the hexadecimal encoded message, ready to send with `cli rest message`
 
+**DEPRECATED**:
```

Comment:
Well no. It gives you the data to for witness

Reply:
My bad, I ment **TO BE DEPRECATED**, or should I remove that notice completely?

Reply:
no **DEPRECATED** is fine. I wrote that in the middle of the night and miss the line I wanted to comment to. It was the line below.